tests/hypervisor/test_local.py

```diff
@@ -5,40 +5,114 @@
 :caseautomation: Automated
 """
 import pytest
-from virtwho import logger
+import time
 
+from hypervisor import logger
 
-class TestLocal:
-    @pytest.mark.tier1
-    def test_hostname_option(self):
-        """Just a demo
 
-        :title: virt-who: local: test hostname option
-        :id: 33cc5ba0-c529-481b-8cfa-8613adbe23ee
+@pytest.mark.usefixtures("function_virtwho_d_conf_clean")
+@pytest.mark.usefixtures("class_debug_true")
+@pytest.mark.usefixtures("class_globalconf_clean")
+class TestLocalPositive:
+    @pytest.mark.tier1
+    def test_threads_number_after_reconfig_libvirt(self, virtwho, ssh_host):
+        """
+        :title: virt-who: loacl: test threads number after reconfig libvirt config file
+        :id: 4d2552ce-af14-4e4f-84ab-64764db64fe2
         :caseimportance: High
         :tags: tier1
         :customerscenario: false
         :upstream: no
         :steps:
-            1.
+            1. Run virt-who service to check the thread number
+            2. Update libvirt config, and restart libvirtd service
+            3. Check virt-who thread_num is changed or not
+            4. Recovery libvirt config
+
         :expectedresults:
-            1.
+            1. Succeeded to start virt-who service and the thread_num is 1
+            2. Succeed to start the libvirtd service
+            3. the thread_num is still 1
         """
-        logger.info("Succeeded to run the 'test_hostname_option'")
 
-    @pytest.mark.tier2
-    def test_http_option(self):
-        """Just a demo
+        # run virt-who service to check the thread number
+        result = virtwho.run_service()
+        thread_berfore = result["thread"]
+        logger.info(
```

Comment:
it seems we should assert the thread_before = 1, not only logger info.

Reply:
I think so, also curious about this when migrating the cases from the old framwork